### DISCUSSION TITLE
Companies & Websites Using Media Chrome

### DISCUSSION THREAD

**Discussion by luwes** [muxinc/media-chrome#760] [Announcements]
If you're using Media Chrome on your website please let us know and share the URL here! Thanks!

**Comment by beingabstrac** [comment]
[60fps](https://www.60fps.design/)

**Comment by tobimori** [comment]
Client project: https://mxr.at/

**Comment by igorvezmar** [comment]
https://player.byomakase.org
Open source frame-accurate player.
Media chrome is used in the player for different use cases (see ‘Use Cases and Demos’).

**Comment by khakra** [comment]
[OctoStream](https://www.octostream.com) - Live Streaming and Re-Streaming Platform
